apps/learning-snippets/src/test/property-grid/ContextMenuItem.test.tsx

```diff
@@ -0,0 +1,107 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+/* eslint-disable no-console */
+/* eslint-disable import/no-duplicates */
+import { expect } from "chai";
+import { join } from "path";
+import sinon from "sinon";
+import { UiFramework, UiItemsManager } from "@itwin/appui-react";
+import { IModelReadRpcInterface, SnapshotIModelRpcInterface } from "@itwin/core-common";
+import { ECSchemaRpcInterface } from "@itwin/ecschema-rpcinterface-common";
+import { ECSchemaRpcImpl } from "@itwin/ecschema-rpcinterface-impl";
+import { PresentationRpcInterface } from "@itwin/presentation-common";
+import { Presentation } from "@itwin/presentation-frontend";
+import { HierarchyCacheMode, initialize as initializePresentationTesting, terminate as terminatePresentationTesting } from "@itwin/presentation-testing";
+// __PUBLISH_EXTRACT_START__ PropertyGrid.ExampleContextMenuItemImports
```

Comment:
Looks like one extract section does not have closing `__PUBLISH_EXTRACT_END__`

Reply:
fixed